app/User.php

```diff
@@ -23,4 +24,9 @@ public function bookings()
     {
         return $this->hasMany(Booking::class);
     }
+
+    public function availability()
```

Comment:
Need to define this new relationship on the user type `schema.graphql`. While you are at it, can you also do it for the bookings and special endorsement relationships? 

See https://lighthouse-php.com/4.8/eloquent/relationships.html#eloquent-relationships

Reply:
done

Reply:
You'll also need to add a type check constraint.

Reply:
done